pkg/policy/validate.go

```diff
@@ -1271,6 +1275,25 @@ func validateResourceDescription(rd kyverno.ResourceDescription) error {
 	return nil
 }
 
+func labelSelectorContainWildcard(v *metav1.LabelSelector) bool {
```

Comment:
minor nit - rename `labelSelectorContainWildcard` to `labelSelectorContainsWildcard`